diff --git a/src/OpenColorIO/transforms/FileTransform.cpp b/src/OpenColorIO/transforms/FileTransform.cpp
--- a/src/OpenColorIO/transforms/FileTransform.cpp
+++ b/src/OpenColorIO/transforms/FileTransform.cpp
@@ -218,6 +218,6 @@ std::unique_ptr<std::istream> getLutData(
 
     // Default behavior. Return file stream.
     return std::unique_ptr<std::istream>(new std::ifstream(
-        Platform::filenameToUTF(filepath), mode));
+        Platform::filenameToUTF(filepath).c_str(), mode));
 }
 
